src/external/monaco.flux.messages.ts

```diff
@@ -41,7 +40,7 @@ const createRequest = (id: number, method: string, params: object = {}) => {
 }
 
 export const initialize = (id: number) => {
-  return createRequest(id, 'initialize')
+  return createRequest(id, 'initialize', {capabilities: {}})
```

Comment:
what else can this be?

Reply:
here are all the options we can provide to `capabilities` https://microsoft.github.io/language-server-protocol/specifications/specification-3-17/#clientCapabilities . The `initialize` [request](https://microsoft.github.io/language-server-protocol/specifications/specification-3-14/#initialize) requires that you at least provide the empty `capabilities` object - so we were not adhering to the LSP spec until now.

For now, to keep us